sherpa/models/basic.py

```diff
@@ -40,14 +40,36 @@
 __all__ = ('Box1D', 'Const1D', 'Cos', 'Delta1D', 'Erf', 'Erfc', 'Exp', 'Exp10',
            'Gauss1D', 'Log', 'Log10', 'LogParabola', 'NormGauss1D', 'Poisson',
            'Polynom1D', 'PowLaw1D', 'Scale1D', 'Sin', 'Sqrt', 'StepHi1D',
-           'StepLo1D', 'Tan', 'Box2D', 'Const2D', 'Delta2D', 'Gauss2D',
-           'SigmaGauss2D',
+           'StepLo1D', 'Tan', 'Voigt1D',
+           'Box2D', 'Const2D', 'Delta2D', 'Gauss2D', 'SigmaGauss2D',
            'NormGauss2D', 'Polynom2D', 'Scale2D', 'UserModel', 'TableModel',
            'Integrate1D')
 
 DBL_EPSILON = numpy.finfo(numpy.float).eps
 
 
+class Voigt1D(RegriddableModel1D):
+    """Voigt1D Profile
+    http://publikationen.badw.de/de/003395768
```

Comment:
@hamogu This reference is in German. Yes, this is the original publication of Voigt profile, but in the documentation we do need to explain the way we implemented it and the meaning of the parameters. We need a proper 'docstring'. I guess this could be done in a separate PR.